friends.md

```diff
@@ -1,5 +1,5 @@
 ### Activities:
-- 2018-11-01: **Grace Hopper** and I went to _Marie's Diner_. George had to cancel at the last minute. @food
+- 2018-11-01: **Grace Hopper** and I went to Marie's Diner. George had to cancel at the last minute. @food
```

Comment:
Maybe: `Marie's Diner on _Martha's Vineyard_`?

Reply:
Ah I see what you're doing here - I take it `Martha's Vineyard` is somewhere I could move to (ie set as a default location)? In which case, we could just change the example to `...and I went to _Martha's Vineyard_`, right? There's no need for `Marie's Diner` because it is not somehwere I could move to and we are already fulfilling the example of a location that is made up of multiple words with `_Martha's Vineyard_`

Reply:
Yes, exactly! Sorry I couldn't think of a better location that fits these criteria. I almost did `St. Kitts` but technically the name is `Saint Kitts` and I wanted to use punctuation too. 🙁

Reply:
Shall I change all instances of `_Marie's Diner`_ to `_Martha's Vineyard_`? Or shall I change them all to `Marie's Diner on _Martha's Vineyard_`?

Reply:
I think either way is fine! Totally your call.